simulation.go

```diff
@@ -0,0 +1,177 @@
+package allocator
+
+import (
+	"encoding/json"
+	"log"
+	"os"
+	"path/filepath"
+	"strconv"
+	"time"
+)
+
+// bazel test ... --test_output=all   --cache_test_results=no   --test_arg='-test.v'   --test_filter='Test.*' --test_env=TEST_TMPDIR='/home/azerila/dev/cap/temp/'
+var dataOutputRoot = filepath.Join("/", "home", "azerila", "dev", "cap", "temp")
```

Comment:
The user would have to manually change this, considering that this path would most likely not be ideal for them